Dockerfile.ci

```diff
@@ -13,5 +13,8 @@ RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 RUN composer install
 COPY --chown=elife .php_cs project_tests.sh smoke_tests.sh ${PROJECT_FOLDER}/
 
+USER root
+RUN create-build-folder
```

Comment:
Should be higher up?

Reply:
Checking out its stability, it can be as it has never changed in a lot of time (just thinking in general terms).